horusec-auth/config/grpc/grpc.go

```diff
@@ -0,0 +1,54 @@
+package grpc
+
+import (
+	"fmt"
+	"github.com/ZupIT/horusec/development-kit/pkg/databases/relational"
+	authGrpc "github.com/ZupIT/horusec/development-kit/pkg/services/grpc/auth"
+	"github.com/ZupIT/horusec/development-kit/pkg/utils/env"
+	"github.com/ZupIT/horusec/development-kit/pkg/utils/logger"
+	"github.com/ZupIT/horusec/horusec-auth/config/app"
+	authController "github.com/ZupIT/horusec/horusec-auth/internal/controller/auth"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials"
+	"net"
+)
+
+func SetUpGRPCServer(postgresRead relational.InterfaceRead, appConfig *app.Config) {
+	if env.GetEnvOrDefaultBool("HORUSEC_GRPC_USE_CERTS", false) {
+		setupWithCerts(postgresRead, appConfig)
+	}
```

Comment:
Isn't necessary an `return` here ?

Reply:
It is not necessary because the function will be looped to receive requests or panic in case of error.